holmes/plugins/toolsets/tempo_tool.py

```
@@ -0,0 +1,270 @@
+                "timestamp": ToolParameter(
+                    description="Timestamp (ISO8601) for the query (e.g., '2025-02-28T00:59:09+05:30')",
+                    type="string",
+                    required=True,
+                ),
+            },
+            toolset=toolset,
+        )
+
+    def _invoke(self, params: Any) -> str:
+        trace_id: str = params["trace_id"]
+        timestamp: str = params["timestamp"]
```

Ruff (0.12.2)

Local variable `timestamp` is assigned to but never used

Remove assignment to unused variable `timestamp`

(F841)